packages/@aws-cdk/aws-amplify/README.md

```diff
@@ -15,8 +15,79 @@
 ---
 <!--END STABILITY BANNER-->
 
-This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+Define a new App:
```

Comment:
Explain what an amplify app is. assume throughout this doc that people don't need to open 100 browser tabs to understand what they need to do. This should be as self-contained as possible.

Reply:
An Amplify App is a collection of branches that represent frontend and backend environments.